client/src/components/styles/_mixin.scss

```diff
@@ -0,0 +1,19 @@
+@mixin mobile-break() {
+  @media screen and (max-width: 393px) {
+    .btn {
+      font-size: 0.75rem;
+    }
+
+    .profile-name {
+      font-size: 1rem;
+    }
+
+    .followers {
+      font-size: 0.8rem;
```

Comment:
0 is not necessary. Lint should be angry for that

Reply:
Lint is not angry)